iosClient/src/components/Matches/MatchedModal.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react';
+import { 
+  StyleSheet,
+  TouchableOpacity,
+  Dimensions,
+  Platform,
+  Switch,
+  Text,
+  View
+} from 'react-native';
+
+import { connect } from 'react-redux';
+
+export class MatchedModal extends Component{
+  constructor (props){
+    super();
+  }
+
+  render() {
+    return (
+      <View style={styles.card}>
+        <View>
+          <Text style={styles.bigText}>{this.props.currentMatch.full_name} wants to meet you too!</Text>
+          <Text style={styles.medText}>Go chat with them now!</Text>
+        </View>
+      </View>
+    )
+  }
+}
+
+const mapStateToProps = (state) => {
+  return {
+    ...state,
+    currentMatch: state.Matches.allMatches,
+  }
+};
+
+export default connect(mapStateToProps)(MatchedModal);
+
+const {width, height} = Dimensions.get('window')
+const styles = StyleSheet.create({
+  card: {
+    flex: 1,
+    height: height*0.8,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: 'white',
```

Comment:
Maybe change all white colors to an off-grey color like Anthony mentioned in his talk